src/Chess.cpp

```diff
@@ -0,0 +1,257 @@
+	#include "Chess.h"
+
+// clear the screen "cls"
+void Chess::clear() const 
+{
+	COORD topLeft = { 0, 0 };
+	HANDLE console = GetStdHandle(STD_OUTPUT_HANDLE);
+	CONSOLE_SCREEN_BUFFER_INFO screen;
+	DWORD written;
+
+	GetConsoleScreenBufferInfo(console, &screen);
+	FillConsoleOutputCharacterA(
+		console, ' ', screen.dwSize.X * screen.dwSize.Y, topLeft, &written
+	);
+	FillConsoleOutputAttribute(
+		console, FOREGROUND_GREEN | FOREGROUND_RED | FOREGROUND_BLUE,
+		screen.dwSize.X * screen.dwSize.Y, topLeft, &written
+	);
+	SetConsoleCursorPosition(console, topLeft);
+}
+// create the GUI - ASCII art
+void Chess::setFrames() 
```

Comment:
There are multiple places where similar logic is repeated. Extracting common logic into separate functions or using loops can make the code more concise and easier to maintain.